src/nl/hannahsten/texifyidea/util/Magic.kt

```diff
@@ -445,40 +441,8 @@ object Magic {
          */
         @JvmField
         val slashlessMathOperators: Set<LatexCommand> = hashSetOf(
```

Comment:
I disagree, I'd keep the long format and quilified names

Reply:
But then you need to do that for everything in Magic. I think it just makes the file unnecessarily long and hard to read.

Reply:
I changed it anyway in #1710